src/App.jsx

```diff
@@ -1,5 +1,14 @@
+import HomeBlog from './Components/HomeBlog';
+import wildHeroes from './Images/wildHeroes.png';
+import './App.css';
+
 function App() {
-  return <h1>Hello there!</h1>;
+  return (
+    <div>
+      <img className="logo" src={wildHeroes} alt="wildHeroes" />
+      <HomeBlog />
+    </div>
+  );
```

Comment:
Everything related to the HomeBlog should be load in the homepage.jsx beside the app.jsx